src/components/inputs/PluginComponentAttributeInputWrapper.vue

```diff
@@ -0,0 +1,43 @@
+<template>
```

Comment:
PluginComponentAttributeInputWrapper` -> `InputWrapper` (could be use somewhere else than plugin component attribute)